Source/EasyNetQ/ProducedMessage.cs

```diff
@@ -10,7 +12,7 @@
         /// </summary>
         /// <param name="properties">The properties</param>
         /// <param name="body">The body</param>
-        public ProducedMessage(MessageProperties properties, byte[] body)
+        public ProducedMessage(MessageProperties properties, ReadOnlyMemory<byte> body)
```

Comment:
The same changes for `ConsumedMessage` ?

Reply:
Done